src/CSSBattle/6_MissingSlice/Yongbin.html

```diff
@@ -0,0 +1,42 @@
+<div class="quad1"></div>
+<div class="quad2"></div>
+<div class="quad3"></div>
+<div class="quad4"></div>
+<style>
+  .quad1 {
+    width: 100px;
+    height: 100px;
+    background: #51b5a9;
+    border-radius: 100px 0 0;
+    position: absolute;
+    top: 50;
+    left: 100;
+  }
+  .quad2 {
+    width: 100px;
+    height: 100px;
+    background: #fade8b;
+    border-radius: 100px 0 0;
+    position: absolute;
+    top: 50;
+    left: 200;
+    transform: rotate(90deg);
+  }
+  .quad3 {
+    width: 100px;
+    height: 100px;
+    background: #f7f3d7;
+    border-radius: 100px 0 0;
+    position: absolute;
+    top: 150;
+    left: 100;
+    transform: rotate(270deg);
+  }
+
```

Comment:
Consider removing unnecessary line break
```suggestion
```

Reply:
Resolved in commit 5ec733b.